documentation/MeetingMinutes/README.MD

```diff
@@ -1 +1,30 @@
-This README.MD can be deleted when the first file is added to this directory.
+# Meeting Details
+
+### Biweekly Meeting
+
+Wednesday ---> 4 PM CET/CEST
+
+### Meeting Link
+
+[Microsoft Meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_MGVkMDUwMTYtNTZiNy00MmMzLWE1M2QtN2Q1ZmEwOWVmZDQ2%40thread.v2/0?context=%7b%22Tid%22%3a%229744600e-3e04-492e-baa1-25ec245c6f10%22%2c%22Oid%22%3a%22ec8dd69b-01fe-4d41-a294-c2927b548e27%22%7d)
+
+### Upcoming Meetings
+
+| Date       |
+|------------|
+| 2023/05/31 |
+| 2023/06/14 |
+| 2023/06/28 |
+| 2023/07/12 |
+| 2023/07/26 |
+| 2023/08/09 |
+| 2023/08/23 |
+
+
+### MoMs
+
+2023/03/22 [Notes](MOM-2023-03-22.md)
+
+2023/04/12 [Notes](MOM-2023-03-22.md)
+
+2023/05/03 [Notes](MOM-2023-05-03.md)
```

Comment:
This will require us to update this README every time we add a meeting minutes document.

Reply:
looks good, thanks.